docs/allocconsole.md

```diff
@@ -5,6 +5,9 @@ author: bitcrazed
 ms.author: richturn
 ms.topic: article
 keywords: console, character mode applications, command line applications, terminal applications, console api
+f1_keywords:
+- consoleapi/AllocConsole
```

Comment:
This one was historically in the wincon.h header and is now in the consoleapi.h header. Should we put both?